src/main/resources/application.yml

```diff
@@ -1,3 +1,26 @@
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    username: postgres
+    password: admin
+    url: jdbc:postgresql://localhost:5432/gd-medical-clinic
+  jpa:
+    hibernate:
+      ddl-auto: validate
+  #USE TO GENERATE SQL SCHEMA SCRIPT FROM JPA ENTITIES
```

Comment:
seems it's not needed anymore?

Reply:
Thought it might be needed in the future, removed it this commit.